crates/bitwarden/src/client/client.rs

```diff
@@ -62,14 +64,15 @@ pub(crate) enum ServiceAccountLoginMethod {
     AccessToken {
         access_token: AccessToken,
         organization_id: Uuid,
+        state_file: Option<PathBuf>,
     },
 }
 
 #[derive(Debug)]
 pub struct Client {
     token: Option<String>,
     pub(crate) refresh_token: Option<String>,
-    pub(crate) token_expires_in: Option<i64>,
+    pub(crate) token_expires_on: Option<i64>,
```

Comment:
Not a huge fan of this name. What about `token_expiration_time`? Follows the https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4.

@dani-garcia Long term do we want to swap out `token` for a JWT object which contains this information instead?

Reply:
Looking a bit into it, I don't think we need the `token` in the root client at all, it only ever seems to be used in `is_authed`, but we already check for `login_method` there anyway.

Same thing with refresh token, which is only used for username login, so it probably doesn't need to be in client either and can be moved into the username `login_method`.

Reply:
Looks like @dani-garcia opened a PR for this here: https://github.com/bitwarden/sdk/pull/431